testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MsalWrapper.java

```diff
@@ -140,12 +140,25 @@ private AcquireTokenParameters.Builder getAcquireTokenParametersBuilder(@NonNull
                 .withPrompt(requestOptions.getPrompt())
                 .withCallback(getAuthenticationCallback(callback));
 
-        // create extra query parameters list and add "is_remote_login_allowed=true"
+        // Create extra query parameters list
+        final List<Map.Entry<String, String>> extraQP = new ArrayList<>();
+
+        // Split given extra query parameters and add them to extra query parameter list
+        // If any were passed in the RequestOptions
+        final String[] requestGenericExtraQueryParams = requestOptions.getExtraQueryParams().split(" ");
```

Comment:
>" " [](http://example.com/codeflow?start=99&length=3)

Would different query params be separated by a whitespaces in UI?

Reply:
you can instead take usual format like key1=value1&key2=value2&key3=value3

Reply:
Yes, similar to scopes from what i understand

![image](https://github.com/user-attachments/assets/d7b86569-f03e-4ae4-9639-ef8ffe65f64d)

Reply:
I can also do &, perhaps that will make things easier if someone is copying from an existing url :)

Reply:
space is more natural to scopes as they are passed that way in the request.

Reply:
Changed this to use & instead of whitespace